packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -125,7 +127,7 @@ const config = async (env): Promise<Configuration> => ({
 
   output: {
     clean: {
-      keep: /gpx_.*/,
+      keep: new RegExp(`gpx_.*|${pluginJson.executable || 'gpx_'}.*`),
```

Comment:
I'm keeping `gpx_.*` for backwards compatibility and intentionally using a redundant fallback. This might be me being too cautious but there's no harm in a duplicated regex or if this might cover some weird edge cases where you have both gpx_ prefixed binaries and custom binaries.

Reply:
This change looks good but I'm not sure it will work with nested plugins. 🤔 

If an App plugin has a `plugin.json` in the root `src` directory but it is the datasource `plugin.json` that declares the executable e.g `../gpx_zabbix-plugin` I think this will continue to fail. Should we worry much? 

`output.clean.keep` can also accept a function. Could we do anything smarter there to understand if the file should be removed or not.

Reply:
@jackw good point on nested plugins.  what if Instead I just check that it doesn't have any of the target platforms suffix? e.g. amd64, arm, etc?

Reply:
Yeah I think that should work. From what I know we append `darwin`, `windows`, and `linux` to whatever is set in the plugin.json `executable` field. 👍